### PR TITLE
Spelling take 3

### PR DESCRIPTION
**Pull request by plaindocs**
This is my third and maybe final attempt at spell checking our stuff.

[Codespell](https://github.com/codespell-project/codespell) looks for known errors, not unknown words, so has less false positives.

1. In the `venv`, install with `pip install codespell`
2. Run using `codespell --skip="*.svg,*.js,*.css,*.scss,docs/_data/201[5|6|7]*" --ignore-words=codespell/ignore.txt docs`

Presumable something like this for the action?

```yaml
  spellcheck:
    runs-on: ubuntu-latest

    steps:
    - uses: actions/checkout@v2
    - name: Set up Python 3.8
      uses: actions/setup-python@v2
      with:
        python-version: 3.8

    - name: Install dependencies
      run: |
        python -m pip install --upgrade pip
        pip install -U codespell

    - name: Check spelling
      run: codespell --skip="*.svg,*.js,*.css,*.scss,docs/_data/201[5|6|7]*" --ignore-words=codespell/ignore.txt docs
```

Can replace #1554

### CODE REVIEW

**Comment by plaindocs** [comment]
FAO @mxsasha @bethaitman @Bradamant3

File: docs/guide/tools/sphinx.rst [hunk 1]
```diff
@@ -144,7 +144,7 @@ The main parts you will need at first are:
 * **Sections**
 * **Directives**
 
-.. note:: You can live-preview RST on the web: http://rst.ninjs.org/
+.. note:: You can live-preview RST on the web: http://rst.ninja.org/
```
Comment (plaindocs) [inline docs/guide/tools/sphinx.rst]:
```suggestion
.. note:: You can live-preview RST on the web: http://rst.ninjs.org/
```

File: codespell/ignore.txt [hunk 2]
```diff
@@ -0,0 +1,12 @@
+ba
+covert
+covert
+crate
+idaes
+ines
+ist
+mobify
```
Comment (plaindocs) [inline codespell/ignore.txt:7]:
```suggestion
mobify
ninjs
```

**Comment by plaindocs** [comment]
Spelling mistake (in this case a false positive) flagged in https://github.com/writethedocs/www/runs/3812433430 

(would be nice to colourise those at some point, but whatever)

File: codespell/ignore.txt [hunk 3]
```diff
@@ -0,0 +1,13 @@
+ba
+covert
+covert
```
Comment (bethaitman) [inline codespell/ignore.txt]: deliberate repeat?

Comment (plaindocs) [reply codespell/ignore.txt]:
```suggestion
```

File: codespell/ignore.txt (hunk 2 above)
Comment (plaindocs) [reply codespell/ignore.txt:7]: @bethaitman the first `covert` was covert, the second `covert` wasn't...

Thanks, deleted.

File: docs/_data/2019.vilnius.speakers.yaml [hunk 4]
```diff
@@ -42,7 +42,7 @@ So far, many technical documents are still very complicated to the point of bein
...
+- abstract: "<p>For the past year, I have been working as one of the small number of tech writers in the Ethereum Blockchain space. As an emerging technology that has existed for only a handful of years, engineers are still figuring out what best practices mean to them. Roles such as tech writer, developer relations and support are even more recent hires, in an ecosystem that fluctuates so much. It's near impossible to remain up to date.\n\n This has put those of us in these roles in a somewhat unique, challenging, and exciting position where we have been able to define what we think \"good\" documentation and communication should look like.\n\n In this presentation, I share experiences (mine and others) from the past year, and techniques and ideas for those who find themselves as the first tech writer hire in a company, or indeed a new field.\n\n The presentation includes:</p>
```
Comment (ericholscher) [inline docs/_data/2019.vilnius.speakers.yaml:45]: Not sure if this is a real issue, but is probably fine.

**Review by ericholscher — APPROVED** [review]
Looks good 👍

**Comment by peternewman** [comment]
> (would be nice to colourise those at some point, but whatever)

We've got an action, it will go one better and do annotations:
https://github.com/codespell-project/actions-codespell

But they should be colourised anyway IIRC.

**Pull request merged**